server/s3_object_metadata.h

```diff
@@ -35,6 +35,8 @@
 #include "s3_request_object.h"
 #include "s3_timer.h"
 
+#define SYM_EXT_OBJ_PREFIX "~"
```

Comment:
have a comment about this character, why its used...

Reply:
ok